package.json

```diff
@@ -60,14 +61,15 @@
     "typescript": "^4.5.2"
   },
   "scripts": {
-    "build": "npm run build:node",
+    "build": "npm run build:node && npm run build:types",
+    "build:types": "tsc -p tsconfig.types.json && cp gen-nodejs/parquet_types.d.ts dist/gen-nodejs/parquet_types.d.ts",
```

Comment:
Why the `cp`? Well thrift generates the js and .d.ts instead of just a .ts file. .d.ts files are considered to be internal only or generated for export only. This odd case requires us to use it as both.